src/tox/tox_env/python/api.py

```diff
@@ -157,6 +157,14 @@ def extract_base_python(cls, env_name: str) -> str | None:
             return next(iter(candidates))
         return None
 
+    @classmethod
```

Comment:
make it private please

Reply:
I will definitively take your lead here, but I wanted to use this in tox-current-env.

Reply:
Making it private ruff-screams at me in tests:

```
tests/tox_env/python/test_python_api.py:314:12: SLF001 Private member accessed: `_python_spec_for_sys_executable`
    |
312 |     mocker.patch.object(sys, "implementation", implementation)
313 |     mocker.patch.object(sys, "maxsize", 2**arch // 2 - 1)
314 |     spec = Python._python_spec_for_sys_executable()
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ SLF001
315 |     assert spec.implementation == impl
316 |     assert spec.major == major
    |

Found 1 error.
```

Reply:
Just silence it. 😅

Reply:
I am unfamiliar with your linter setup (it feels like an obstacle rather than something that would help me as a drive-by contributor). Could you please tell me what exactly is necessary or better yet push a commit here that does that? I could go and learn how to configure ruff to exclude tests from "Private member accessed" checked, but I assume you know this better than me.

Reply:
# noqa: SLF001